nncf/config/schemata/algo/filter_pruning.py

```diff
@@ -0,0 +1,162 @@
+                   f"See [Pruning.md]"
+                   f"({ONLINE_DOCS_ROOT}"
+                   f"/docs/compression_algorithms/Pruning.md) and the rest of this schema for "
+                   f"more details and parameters."
+    ,
+    "properties": {
+        "algorithm": {
+            "const": FILTER_PRUNING_ALGO_NAME_IN_CONFIG
+        },
+        "initializer": GENERIC_INITIALIZER_SCHEMA,
+        "pruning_init": with_attributes(NUMBER,
+                                        description="Initial value of the pruning level applied to the "
+                                                    "prunable operations.",
+                                        default=PRUNING_INIT),
+        "params":
+            {
+                "type": "object",
+                "properties": {
+                    "filter_importance": with_attributes(STRING,
+                                                         description="The type of filter importance metric.",
+                                                         enum=FILTER_IMPORTANCE_OPTIONS,
+                                                         default=PRUNING_FILTER_IMPORTANCE),
+                    "pruning_target": with_attributes(NUMBER,
+                                                      description="Target value of the pruning level for "
+                                                                  "the operations that can be pruned. "
+                                                                  "The operations are determined by the model "
+                                                                  "architecture as analyzed by the algorithm.",
+                                                      default=PRUNING_TARGET),
+                    "pruning_steps": with_attributes(NUMBER,
+                                                     description="Number of epochs during which the pruning level is "
+                                                                 "increased from `pruning_init` to `pruning_target`.",
+                                                     default=PRUNING_STEPS),
+                    "pruning_flops_target": with_attributes(NUMBER,
+                                                            description="Target value of the pruning level for model "
+                                                                        "FLOPs."),
+                    "schedule": with_attributes(STRING,
+                                                description="The type of scheduling to use for adjusting the target "
+                                                            "pruning level. ",
+                                                enum=FILTER_PRUNING_SCHEDULE_OPTIONS,
+                                                default=PRUNING_SCHEDULE),
+
+                    "num_init_steps": with_attributes(NUMBER,
+                                                      description="Number of epochs for model pretraining before "
```

Comment:
```suggestion
                                                      description="A number of epochs for model pretraining before "
```

Reply:
Articles are often omitted in technical writing.